src/components/pokemonCard/PokemonCard.jsx

```diff
@@ -0,0 +1,74 @@
+import './PokemonCard.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import ErrorMessage from "../errorMessage/ErrorMessage.jsx";
+
+const PokemonCard = ({ endpoint }) => {
+    const [poke, setPoke] = useState({});
```

Comment:
Ik zou als default waarde `undefined` nemen. Dan is het wat explicieter dat het nog ingeladen moet worden en kan je check verderop iets simpeler
```suggestion
    const [poke, setPoke] = useState();
```